schemas/com.snowplowanalytics.accelerators.travel/destination_filter/jsonschema/1-0-0

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "type": "object",
+  "self": {
+    "vendor": "com.snowplowanalytics.accelerators.travel",
+    "name": "destination_filter",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Event for filtering destination specific information",
+  "properties": {
+    "page_type": {
+      "type": "string",
+      "description": "Page type"
```

Comment:
I noticed this doesn't have a max length, but most of your other string fields have a max length.